docs/docsite/rst/playbook_guide/playbooks_checkmode.rst

```diff
@@ -68,7 +68,7 @@ If you want to skip a task or ignore errors on a task when you run Ansible in ch
         dest: /home/mylogin/hello
       when: not ansible_check_mode
 
-    - name: This task will ignore errors in check mode
+    - name: This task will ignore errors in check mode, but also outside of check mode because of https://github.com/ansible/ansible/issues/56345
```

Comment:
Hmm. This is a little hard to read and seems out of place. Putting a url inline also gets the user off task.

How about adding this as a note immediately after the codeblock? That way you could give a bit more context as well as give the issue some additional highlighting?